api/src/db.py

```diff
@@ -0,0 +1,20 @@
+import os
+from collections.abc import Generator
+from typing import Any
+
+from dotenv import find_dotenv, load_dotenv
+from sqlmodel import Session, create_engine
+
+load_dotenv(find_dotenv())
+
+DATABASE_URL: str = os.environ["DATABASE"]
+engine = create_engine(DATABASE_URL, echo=False)
+session = Session(bind=engine, autoflush=True)
```

Comment:
The session object used globally in this app has `autoflush` enabled. In `classes/company.py` there's a few calls of `session.flush()` before commiting to the db. Are the explicit flush calls are necessary given that autoflush is already enabled?

Reply:
Sometimes I don't always trust autoflush (or forget about those). Can remove them.